contentcuration/contentcuration/viewsets/contentnode.py

```diff
@@ -556,6 +557,16 @@ def requisites(self, request, pk=None):
             ),
         )
 
+    @detail_route(methods=["get"])
+    def size(self, request, pk=None):
+        if not pk:
+            raise Http404
+        node = self.get_object()
+        files = File.objects.filter(contentnode__in=node.get_descendants(include_self=True)).values("checksum").distinct()
```

Comment:
As far as I see this is including the incomplete resources that will not be published. If that's fine, perfect

Reply:
I think it would make sense to update that, this was copied from the original implementation.

The other slight issue is that this is not an accurate representation of the eventual size of the published channel, as it does not include the size of the exercises.